src/main/java/uk/ac/ebi/atlas/hcalandingpage/HcaMetadataController.java

```diff
@@ -0,0 +1,30 @@
+package uk.ac.ebi.atlas.hcalandingpage;
+
+import com.google.gson.JsonObject;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import static uk.ac.ebi.atlas.utils.GsonProvider.GSON;
+
+@RestController
+public class HcaMetadataController {
+    private final static String SPECIES = "Homo sapiens";
```

Comment:
A constant’s name should mean something specific. E.g.:
```java
// Good
private final static double PI = 3.1415926535
// Not good
private final static double DOUBLE = 3.1415926535
```

Reply:
Removed it as it was being used at one place in the code so removed it.